### PR TITLE
rocket explosion vars and cleanup

### PR DESCRIPTION
**Pull request by SonixApache**
- Rockets now have vars to determine their EMP/explosion size, so admins can fuck with them if they want.
- Did some cleanup, removed most of the copypaste. Woohoo, OOP.
- Nikitas are now a subtype of rockets.
- Added a maxcap meme rocket.

Did you know that if you fire any RPG at a reinforced wall, there's a pretty high chance it won't even pierce the wall? Hitting walls with RPGs is completely fucked, man. I'm not gonna be the one to fix it, though.

### CODE REVIEW

**Comment by FudgePucker** [comment]
I mean, plasteel is magic. Plasma is a magical mcguffin

**Comment by SonixApache** [comment]
Hitting someone in front of the plasteel wall nukes the wall behind them and some other ones.
Hitting the plasteel wall itself eats half the explosion.

**Comment by FudgePucker** [comment]
Oh, that does sound broken

File: code/modules/projectiles/projectile/rocket.dm [hunk 1]
```diff
@@ -39,77 +45,55 @@
...
 /obj/item/projectile/rocket/stun/to_bump(var/atom/A)
-	explosion(A, -1, 0, 0, 0)
 	flashbangprime(TRUE, FALSE, FALSE)
 	..()
-	if(!gcDestroyed)
-		qdel(src)
-
-/obj/item/projectile/nikita
+		
+/obj/item/projectile/rocket/lowyield/extreme
```
Comment (Kurfursten) [inline code/modules/projectiles/projectile/rocket.dm:88]: >low yield

big hmm

Comment (SonixApache) [reply code/modules/projectiles/projectile/rocket.dm:88]: it's already coded/mapped in and i didn't feel like changing it and that specific subtype explodes

**Pull request merged**